packages/integrations/node/package.json

```diff
@@ -19,8 +19,8 @@
   "homepage": "https://docs.astro.build/en/guides/integrations-guide/node/",
   "exports": {
     ".": "./dist/index.js",
-    "./server.js": "./dist/server.js",
-    "./preview.js": "./dist/preview.js",
+    "./server.js": "./dist/server-entrypoint.js",
+    "./preview.js": "./dist/preview-entrypoint.js",
```

Comment:
This is the only change I don't love! I think we should probably remove the `-entrypoint` suffix so that the exported path and the actual file path continue to match.

Reply:
This was mostly because every other file had its own value that it called a "server", but yeah for entrypoints, it may have been an over-correction.